NEWS.md

```diff
@@ -1,5 +1,18 @@
 # protti 0.7.0.9000
 
```

Comment:
Shouldnt we release protti 0.8.0 0r 0.7.1? Since we are adding several new features https://semver.org/

Reply:
Yes, I would make it protti 0.8.0 because we introduce some new features. During development I keep the version number like this to indicate that it is not yet a new release. Before releasing it I change it to the new release version number.

Reply:
Ah okay nice!